public/app/features/playlist/PlaylistPage.tsx

```diff
@@ -51,17 +62,26 @@ export const PlaylistPage: FC<Props> = ({ navModel }) => {
                 Start playlist
               </Button>
               {contextSrv.isEditor && (
-                <LinkButton key="edit" variant="secondary" href={`/playlists/edit/${playlist.id}`} icon="cog">
-                  Edit playlist
-                </LinkButton>
+                <>
+                  <LinkButton key="edit" variant="secondary" href={`/playlists/edit/${playlist.id}`} icon="cog">
+                    Edit playlist
+                  </LinkButton>
+                  <Button
+                    disabled={false}
+                    onClick={() => setPlaylistToDelete({ id: playlist.id, name: playlist.name })}
```

Comment:
nit: anonymous function

Reply:
But we need to pass in an argument available only in the context. How can achieve this without the anonymous function?

Reply:
Good catch, I didn't pay attention to the arguments. You can ignore my comment.